receiver/hostmetricsreceiver/testdata/config.yaml

```diff
@@ -0,0 +1,17 @@
+receivers:
+  hostmetrics:
+    scrape_interval: 10s
+    scrapers:
+
+processors:
+  exampleprocessor:
+
+exporters:
+  exampleexporter:
+
+service:
+  pipelines:
+    traces:
```

Comment:
should be metrics

Reply:
Addressed these your comments in the next PR: https://github.com/open-telemetry/opentelemetry-collector/pull/862